<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotDashboard</class>
 <widget class="QMainWindow" name="RobotDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>793</width>
    <height>593</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Underwater Robot Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>793</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="VideoFeed">
     <attribute name="title">
      <string>Video Feed</string>
     </attribute>
     <widget class="QWidget" name="CameraMenu" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>791</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: gray;</string>
      </property>
      <widget class="QComboBox" name="CameraBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>31</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Camera">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>31</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Graphs">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QWidget" name="RecordMenu" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>791</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;</string>
      </property>
      <widget class="QPushButton" name="attachFile">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string>Attach File</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
